internal/v1/handler_blueprints.go

```diff
@@ -87,6 +87,13 @@ func (bb *BlueprintBody) RedactCertificates() {
 	bb.Customizations.Cacerts = nil
 }
 
+func (bb *BlueprintBody) RedactAAPRegistration() {
+	if bb.Customizations.AAPRegistration != nil {
+		bb.Customizations.AAPRegistration.TlsCertificateAuthority = ""
```

Comment:
TLS Certificate is not a secret, the secret part is a key that is not stored in this field. You may delete this line.